src/main/java/org/openpnp/machine/neoden4/wizards/Neoden4CameraConfigurationWizard.java

```diff
@@ -46,8 +52,8 @@ public Neoden4CameraConfigurationWizard(Neoden4Camera camera) {
 
         panelGeneral = new JPanel();
         contentPanel.add(panelGeneral);
-        panelGeneral.setBorder(new TitledBorder(null,
-                "General", TitledBorder.LEADING, TitledBorder.TOP, null));
+        panelGeneral.setBorder(new TitledBorder(new EtchedBorder(EtchedBorder.LOWERED, null, null),
```

Comment:
Sorry, I missed these so far. You should search all your sources for `EtchedBorder `and get rid of them.

See #1289.